docs/config.md

```diff
@@ -309,3 +309,53 @@ spec:
 Each `configMapName` must refer to the name of a Config Map in the same namespace as Cryostat. The corresponding `filename` must be a key within that Config Map containing resource mappings. The `clusterRoleName` must be a valid name of an existing Cluster Role.
 
 **Note:** If the mapping is updated, Cryostat must be manually restarted.
+
+
+### Security Context
+
+With [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/), pods that are not properly configured under the enforced security standards defined globally or on namspace level will be admitted to launch.
+
+The user is responsible for ensuring the security contexts of their workloads to meet these standards. The property `spec.securityOptions` can be set to define security contexts for Cryostat application and its report sidecar.
```

Comment:
You might want to reference `spec.reportOptions.securityOptions` for the latter.

Reply:
We might also want to mention the defaults we use. The defaults conform to the `restricted` Pod Security Standard [1]. For the Cryostat application pod, the operator selects an `fsGroup` to ensure that Cryostat can read and write files in its Persistent Volume.

On OpenShift, `spec.securityContext.seccompProfile` is left unset for backward compatibility. On versions of OpenShift supporting Pod Security Admission, the `restricted-v2` Security Context Constraint sets `seccompProfile` to `runtime/default` as required for the `restricted` Pod Security Standard. [2]

[1] https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
[2] https://docs.openshift.com/container-platform/4.11/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth

Reply:
Right! Theses are really important! I will add these!